index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require 'src/db.php';
+require 'src/functions.php';
+
+$db = connectToDb('artistsApp');
+$artists = getAllArtists($db);
+
+
+?>
+
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Chloe's Collector App</title>
+    <link rel="stylesheet" href="https://use.typekit.net/rkk2hbc.css">
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+    <html lang="en"></html>
+</head>
+    <header>
+        <h1 tabindex=0 >ARTISTS</h1>
+    </header>
+<body>
+    <?php
+    foreach($artists as $artist) {
+        try {
+            $artistsOutput = generateArtistHtml($artist);
+            echo $artistsOutput;
```

Comment:
This isn't a change request, this bit of code is totally fine as it is.

Some would say that creating the variable here isn't needed as you're only using the results of `generateArtistHtml()` a single time - you could just do `echo generateArtistHtml($artist);`

Like I said, not something you really need to change, I'd accept it as is, just one to note!

Reply:
Changed to one line!